kandy-geo.json

```diff
@@ -0,0 +1,30 @@
+{
+  "description": "Kotlin plotting DSL for Lets-Plot",
+  "properties": [
+    {
+      "name": "kandyVersion",
+      "value": "0.8.0"
+    },
+    {
+      "name": "kandyVersion-renovate-hint",
+      "value": "update: package=org.jetbrains.kotlinx:kandy-geo"
+    },
+    {
+      "name": "dataframeVersion",
+      "value": "0.15.0"
+    },
+    {
+      "name": "dataframeVersion-renovate-hint",
+      "value": "update: package=org.jetbrains.kotlinx:dataframe-geo"
+    }
+  ],
+  "link": "https://github.com/Kotlin/kandy",
+  "repositories": [
+    "https://repo.osgeo.org/repository/release"
+  ],
+  "dependencies": [
+    "org.jetbrains.kotlinx:dataframe-geo:$dataframeVersion",
+    "org.jetbrains.kotlinx:kandy-geo:$kandyVersion",
+    "org.apache.logging.log4j:log4j-core:2.20.0"
```

Comment:
Is it necessary?

Reply:
I mean log4j

Reply:
Yep, required by geotools. Without it there are debug lines in outputs.